glean/load.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// clang-format off
+// This file exposes a list of metrics/pings to the QML/JS world.
+.pragma library
+// Importing Glean.js as released by the Glean team.
+.import 'glean.js' as RealGlean
+// clang-format on
+
+const _glean = RealGlean.Glean.default;
+
+// The internal ping to send the pending samples.
+const _ping = new _glean._private.PingType(
+    {
+      includeClientId: true,
+      sendIfEmpty: false,
+      name: 'ping',
+      reasonCodes: [],
+    },
+);
+
+// List of exposed samples
+// TODO: write a proper telemetry metrix document. The following sample is just
+// for testing.
+const appStarted = new _glean._private.DatetimeMetricType(
+    {
+      category: 'sample',
+      name: 'app_started',
+      sendInPings: ['ping'],
+      lifetime: 'ping',
+      disabled: false,
+    },
+    'millisecond');
+
+// public functions
+
+let _initialized = false;
+function initialize(appVersion, uploader) {
+  if (!_initialized) {
+    _initialized = true;
+    _glean.initialize('MozillaVPN', true, {
```

Comment:
It's not ideal not pass a hard coded value for the `uploadEnabled` status, but rather the user's preference on whether or not telemetry should be enabled.